src/map/battle.c

```diff
@@ -4716,6 +4710,12 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 	}
 	if (flag.cri) {
 		wd.type = BDT_CRIT;
+#ifdef RENEWAL
+#if PACKETVER >= 20161207
```

Comment:
this checking, maybe could be removed? since we could check if the setting are enabled or not during startup.